### DISCUSSION TITLE
Pre-deployment webhooks (Per team, or system wide)

### DISCUSSION THREAD

**Discussion by maietta** [coollabsio/coolify#2778] [Feature Requests]
I'm looking for a clean way to receive notification when a deployment has begun, even though it has not finished.

This would allow my system to update the DNS records to point to wherever the resources are being deployed to. I would at minimum, want to know 1) The domains being deployed for 2)  the server(s) being deployed to.

I would like to configure this webhook at the team or system-wide level. I think teams makes more sense, because the servers are divided among teams.

**Comment by harshverma250202** [comment]
any progress made on this?

**Comment by bobbymannino** [comment]
This would be incredibly useful if it was project or even application specific
